crates/broker/src/order_monitor.rs

```diff
@@ -163,14 +174,17 @@ where
         // TODO: spawn a side thread to avoid missing new blocks while this is running:
         let order_count = if let Some(last_monitor_block) = opt_last_block {
             let current_block = self.chain_monitor.current_block_number().await?;
+            let current_block_timestamp = self.chain_monitor.current_block_timestamp().await?;
 
+            // TODO: It looks like this might now be true.
```

Comment:
Is this just a note that this is a fix? Can remove

Reply:
Ah, I forgot to come back to that. I now understand what it's doing here, and I've adjusted the message to be more accurate. I think at some point the logic must have changed here. There still may be value in actually trying to scan the chain for locked orders that are _not_ in the DB, as a way of recovering from error.

c23799f0685619bf10757ea02de9a23e95498472